### PR TITLE
[Guidos Gorgeous Lasagna] Rewrites

### PR DESCRIPTION
**Pull request by BethanyG**
Another round of rewrites.

1.  More shortening of both exercise and concept `introduction.md`
2. Additional edits to concept `about.md`
3. Additional edits to stub file
4. Additional edits to the `instruction.md` file

### CODE REVIEW

**Comment by BethanyG** [comment]
@iHiD @kytrinyx  - apologies for pining you both here.  There is no huge rush on these, but I definitely need some perspective, because I am beginning to doubt my writing and programming skills. I think this is the last rewrite I'll be doing , at least for a while.  Many many thanks in advance for any 👀  you can lend.

**Comment by kytrinyx** [comment]
@BethanyG Sorry it's taken so long to get to this. I _think_ this all looks like good changes. I definitely agree that shortening things up a bit in the start there is a good choice.

I would say... ship-it. If people are confused, they'll open a thread in the forum and that will be great information that we can act on later.

**Comment by BethanyG** [comment]
@kytrinyx   💜   Thanks - [there is a thread on the forum](https://forum.exercism.org/t/i-am-struggling-with-the-lasagna-exercise/2091), but I don't know that I agree with everything suggested.  This was my interpretation, hence the request for review from you and Jeremey.  But if this looks ok, I'll merge it once I get an approval.  Beyond that, maybe the influx of students for Analytical April will prompt additional complaints and suggestions....

File: exercises/concept/guidos-gorgeous-lasagna/.docs/introduction.md
Comment (iHiD) [inline exercises/concept/guidos-gorgeous-lasagna/.docs/introduction.md:3]:
```suggestion
Python is a [dynamic and strongly typed][dynamic typing in python] programming language.
```

File: exercises/concept/guidos-gorgeous-lasagna/.docs/introduction.md
Comment (iHiD) [inline exercises/concept/guidos-gorgeous-lasagna/.docs/introduction.md:9]: Nice.

File: exercises/concept/guidos-gorgeous-lasagna/.docs/introduction.md
Comment (iHiD) [inline exercises/concept/guidos-gorgeous-lasagna/.docs/introduction.md:11]: Also nice.

File: exercises/concept/guidos-gorgeous-lasagna/.docs/introduction.md
Comment (iHiD) [inline exercises/concept/guidos-gorgeous-lasagna/.docs/introduction.md:27]:
```suggestion
Programmer-defined [_names_][facts-and-myths-about-python-names] (also called _variables__) can be bound to any type of object using the assignment `=` operator: `<name> = <value>`.
```

File: exercises/concept/guidos-gorgeous-lasagna/.docs/introduction.md
Comment (iHiD) [inline exercises/concept/guidos-gorgeous-lasagna/.docs/introduction.md:32]:
```suggestion
>>> my_first_variable = 1  # my_first_variable bound to an integer object of value one.
```

File: exercises/concept/guidos-gorgeous-lasagna/.docs/introduction.md
Comment (iHiD) [inline exercises/concept/guidos-gorgeous-lasagna/.docs/introduction.md:33]:
```suggestion
>>> my_first_variable = 2  # my_first_variable re-assigned to integer value 2.
```

File: exercises/concept/guidos-gorgeous-lasagna/.docs/introduction.md
Comment (iHiD) [inline exercises/concept/guidos-gorgeous-lasagna/.docs/introduction.md:65]:
```suggestion
# The body of a function is indented by 2 spaces, & prints the sum of the numbers
```

**Review by iHiD — APPROVED** [review]
I really like these changes. Nice work :) I've made a few comments just for where things weren't immediately clear to me, in the hope of making them clearer to others. But you judge whether they're good suggestions or not :)

**Comment by iHiD** [comment]
I didn't see this until now. Sorry.

This is the current state of my GH notifications:

Screenshot 2023-03-03 at 18 47 08

@BethanyG Definitely always feel free to ping me on Slack or on the forum 1-1 if you'd like me to review something as there's a almost certain chance I'll miss things via my notifications! Always happy to be of help to you if I can 💙

File: exercises/concept/guidos-gorgeous-lasagna/.docs/introduction.md
Comment (BethanyG) [reply exercises/concept/guidos-gorgeous-lasagna/.docs/introduction.md:3]: _thank you_

File: exercises/concept/guidos-gorgeous-lasagna/.docs/introduction.md
Comment (BethanyG) [reply exercises/concept/guidos-gorgeous-lasagna/.docs/introduction.md]: Resolved this one on the branch.

File: exercises/concept/guidos-gorgeous-lasagna/.docs/introduction.md
Comment (BethanyG) [reply exercises/concept/guidos-gorgeous-lasagna/.docs/introduction.md]: Thank you.  This reads much better now!

**Comment by BethanyG** [comment]
Whelp, let's hope this helps.  The stats have now sunk to 42.99% from 43.1%!!  😱 .  Thanks again to the both of you for the review! 💜

**Pull request merged**
